johnsonandjohnson/Bodiless-JS
# fix(test-site): Reusable Logo

## Changes

- Change Logo to reusable between deskheader and mobile burger menu
- Allows site logo to be specified in gatsbyconfig.js and updating the logo there changes it both places 
- Renamed ExampleHelmet -> SiteHelmet because example didn't make sense.
- Split out design for header/footer/logo to token.ts
- Renamed index.jsx -> index.tsx

## Test Instructions
* On Static, Edit & Edit preview -- verify header, logo, footer and burger-menu all operate as before. 
There is slight change in padding for burger menu to give space around the burger icon.

## Related Issues
